src/main/java/Calculator/Operator.java

```diff
@@ -0,0 +1,35 @@
+package Calculator;
+
+import java.util.function.BiFunction;
+
+public enum Operator {
+    ADD("+", (first, second) -> first + second),
+    SUBTRACT("-", (first, second) -> first - second),
+    MULTIPLY("*", (first, second) -> first * second),
+    DIVIDE("/", (first, second) -> {
+        if (second == 0) {
+            throw new ArithmeticException("0으로 나눌 수 없습니다.");
+        }
+        return first / second;
+    });
+
+    private String symbol;
+    private BiFunction<Integer, Integer, Integer> expression;
+
+    Operator(String symbol, BiFunction<Integer, Integer, Integer> expression) {
+        this.symbol = symbol;
+        this.expression = expression;
+    }
+    public int operate(int first, int second) {
+        return expression.apply(first, second);
+    }
+
+    public static Operator findOperator(String symbol) {
+        for (Operator operator: Operator.values()) {
+            if (operator.symbol.equals(symbol)) {
+                return operator;
+            }
+        }
+        throw new IllegalArgumentException("+, -, *, / 연산자가 아닙니다.");
```

Comment:
```suggestion
		return Arrays.stream(Operator.values())
			.filter(operator -> operator.symbol.equals(symbol))
			.findFirst()
			.orElseThrow((() -> new IllegalArgumentException("사칙연산 기호가 아닙니다.")));
```

반복문도 충분히 좋지만 Stream을 한번 학습할 겸 사용해보는 것도 좋을 것 같습니다.
알아두면 좋은 api들이 많고 많이 활용되기 때문이죠.!